modules/azurerm-eventhub-namespace-authorization-rule/main.tf

```diff
@@ -0,0 +1,24 @@
+resource "azurerm_eventhub_namespace_authorization_rule" "this" {
+  lifecycle {
+    # precondition {
```

Comment:
Is this meant to stay in here?

Reply:
Was an attempt to ensure atleast 1 was set but I clearly gave up and never came back to it sooooo nah, will leave a todo for it.